src/app/app.component.html

```diff
@@ -4,8 +4,25 @@
 </mo-header>
 
 <ng-container *ngIf="weatherData.length==locLength">
-  <ng-container *ngFor="let data of weatherData; index as i">
-    <button (click)="locationID = i">{{data.location.name}}</button>
-  </ng-container>
-  <app-weather-data *ngIf="locationID || locationID==0" [data]="weatherData[locationID]"></app-weather-data>
+  <div id="buttonDiv">
+    <ng-container *ngFor="let data of weatherData; index as i">
+      <button (click)="locationID = i"><b>{{data.location.name}}</b></button>
+    </ng-container>
+  </div>
+
+  <div id="smallSelect">
+    <mat-form-field>
+      <mat-select class="drop">
```

Comment:
I'd probably give this one and the class on the option a more descriptive name, so it's a bit clearer when you're reading the CSS what it's for, 
the bundling and scoping of the app will minify these (& basically everything else) for you during the build, so descriptive names are always better for readbility 👍